apiary-metastore-events/README.md

```diff
@@ -0,0 +1,17 @@
+# Apiary Metastore Events
+
+##  Overview
+The Apiary Metastore Events contains set of modules responsible for retrieving and processing of the Hive Metastore events.
+
+Currently, the following modules are defined:
+ - [apiary-metastore-listener](apiary-metastore-listener) - Listens to the events from Hive Metastore and publishes them to SNS
+ - [apiary-receivers](apiary-receivers) - Provides an implementation for polling SQS messages
```

Comment:
I think the description for `apiary-receivers` can be improved to say "provides an implementation for receiving and deserializing hive events from a queue".

I know it doesn't say that on the README for `apiary-receivers` at the moment. Perhaps that should be improved too later.